kustomize-fns/v1alpha1/applyresources/ApplyResources.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/discovery/cached/memory"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/restmapper"
+	"os"
+	"sigs.k8s.io/kustomize/v3/k8sdeps/kunstruct"
+	"sigs.k8s.io/kustomize/v3/k8sdeps/transformer"
+	"sigs.k8s.io/kustomize/v3/pkg/ifc"
+	"sigs.k8s.io/kustomize/v3/pkg/resmap"
+	"sigs.k8s.io/kustomize/v3/pkg/resource"
+	"sigs.k8s.io/kustomize/v3/pkg/transformers/config"
+	"sigs.k8s.io/kustomize/v3/pkg/types"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	// configurableResourcesLabel is a label added by odh dev to specify which objects are can be updated.
+	configurableResourcesLabel = "opendatahub.io/configurable"
+	// modifiedResourcesLabel is a label added by end user to specify a given resource has been updated.
+	modifiedResourcesLabel     = "opendatahub.io/modified"
+	// forceUpdateResourcesLabel is a label added by odh devs to specify that an object needs to be overwritten
+	// irrespective of the modified label.
+	forceUpdateResourcesLabel  = "opendatahub.io/force-update"
+)
```

Comment:
What is the use case for force-update?

Reply:
`force-update` can be applied during upgrades. If we want the user modified config to get newer changes, `force-update` label can be added to the manifests. It will rewrite the object even if it has `modified` label